src/domain/Money.js

```diff
@@ -0,0 +1,53 @@
+import ERROR_MESSAGE from '../constants/errorMessage.js';
+import { MONEY_UNIT } from '../constants/lottoStandard.js';
+import CustomError from '../CustomError.js';
+
+class Money {
+  #money;
+
+  constructor(money) {
+    this.#validate(money);
+    this.#money = Number(money);
+  }
+
+  #validate(money) {
+    this.#validateNotEmpty(money);
+    this.#validateNumberType(money);
+    this.#validateInBoundary(money);
+    this.#validateDivideByBasicMoney(money);
+  }
+
+  #validateNotEmpty(money) {
+    if (money === null || money === '') {
+      throw new CustomError(ERROR_MESSAGE.EMPTY_MONEY);
+    }
+  }
```

Comment:
```suggestion
  #validateNotEmpty(money) {
    if (!money) {
      throw new CustomError(ERROR_MESSAGE.EMPTY_MONEY);
    }
  }
```

로 바꾸면 조건식을 하나로 줄일 수 있을 것 같습니다!

Reply:
와! 이렇게 줄어들 수 있었군요..
코드 첨부까지 자세한 피드백 감사합니다 👍👍